### PR TITLE
add reshape/subset/fields snippet

### PR DESCRIPTION
**Pull request by ddebrunner**
Adds the first set of snippets related to reshaping

### CODE REVIEW

File: reshape/fields/config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  policies:
+    - type: Query
+      rules:
+        - condition: true
+          name: api
+          fields: [hominoid, person, defaultHero, luke, robot]
+        - condition: true
+          name: introspection
+          fields: [__schema, __type, __typename]
+      policyDefault:
```
Comment (royderks) [inline reshape/fields/config.yaml:11]: Some comments in this file will help understanding the mechanics of specifically defining the fields that are exposed would be in place. Especially, as users would expect all fields to be exposed by default

File: reshape/subset/config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+access:
+  policies:
+    - type: Query
+      policyDefault:
```
Comment (royderks) [inline reshape/subset/config.yaml:4]: Is this file needed as it takes the default of exposing all fields? Or did you put it in here on purpose to reduce confusion for users trying out the other reshape examples?

File: reshape/subset/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Deploy the schema from `reshape/subset` relative to the repository's root directory:
+
+```
+stepzen deploy
+```
+
+Run the [sample operations](operations.graphql):
+
+```
+stepzen request -f operations.graphql Human -H Authorization:
```
Comment (royderks) [inline reshape/subset/README.md:79]: `--operation-name` is missing here 

>  ›   Error: Unexpected flag -f, --file=operations.graphql. Could not handle both the QUERY argument and the -f, --file flag. Please remove one.

**Review by royderks — COMMENTED** [review]
Nice work, thanks for the extensive documenting in the READMEs as well. This will help creating the docs for the website

File: reshape/subset/config.yaml (hunk 2 above)
Comment (ddebrunner) [reply reshape/subset/config.yaml:4]: It's needed as for all the examples I made the statement that they would be accessible without authorization. Without this file then authorization is needed.

File: reshape/subset/README.md (hunk 3 above)
Comment (ddebrunner) [reply reshape/subset/README.md]: Fixed.

**Review by royderks — APPROVED** [review]
